products.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+require_once 'src/Factories/PdoFactory.php';
+require_once 'src/Models/ProductModel.php';
+require_once 'src/Services/ProductListDisplayService.php';
+require_once 'src/Models/CategoryModel.php';
+require_once 'src/Entities/Product.php';
+
+$category = false;
+$productList = [];
+$db = PdoFactory::connect();
+
+if (isset($_GET['cat_id']) && is_numeric($_GET['cat_id'])) {
+        $cat_id = intval($_GET['cat_id']);
+        $category = CategoryModel::getCategoryById($cat_id, $db);
+        if ($category){
+            $categoryTitle = $category->getCategoryName();
+            $productList = ProductModel::getProductByCatId($cat_id, $db);
+        }
+}
+
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Furniture Store</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body>
+<nav class="bg-slate-800 py-2 px-5">
+    <span class="text-4xl text-white">Furniture Store</span>
+</nav>
+    <div class="container mx-auto md:w-2/3 mt-5">
+    <?php
+    if ($category) {
+        echo ProductListDisplayService::displayCategoryTitle($categoryTitle);
+        }
+    else {
```

Comment:
```suggestion
    } else {
```

Indentation and placement of `else`